src/wp-includes/class-wp-script-modules.php

```diff
@@ -166,6 +166,7 @@ public function add_hooks() {
 		add_action( $position, array( $this, 'print_import_map' ) );
 		add_action( $position, array( $this, 'print_enqueued_script_modules' ) );
 		add_action( $position, array( $this, 'print_script_module_preloads' ) );
+		add_action( 'wp_footer', array( $this, 'print_import_map_polyfill' ), 11 );
```

Comment:
Why isn't it using `$position` like other actions?

Reply:
In this case the polyfill is always working in the footer, it has not theme-type conditional like modules. It's more optimised being on the footer, not too much, but optimised.